src/wp-includes/block-editor.php

```diff
@@ -303,21 +303,38 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 		$custom_settings
 	);
 
-	$theme_json = WP_Theme_JSON_Resolver::get_merged_data();
+	$presets = array(
+		array(
+			'css'                     => 'variables',
+			'__unstableType'          => 'presets',
+			'__experimentalNoWrapper' => true,
+		),
+		array(
+			'css'            => 'presets',
+			'__unstableType' => 'presets',
+		),
+	);
+	foreach ( $presets as $preset_style ) {
+		$actual_css = wp_get_global_stylesheet( array( $preset_style['css'] ) );
+		if ( '' !== $actual_css ) {
+			$preset_style['css']         = $actual_css;
+			$editor_settings['styles'][] = $preset_style;
+		}
+	}
 
 	if ( WP_Theme_JSON_Resolver::theme_has_support() ) {
-		$editor_settings['styles'][] = array(
-			'css'            => $theme_json->get_stylesheet( array( 'styles', 'presets' ) ),
-			'__unstableType' => 'globalStyles',
-		);
-		$editor_settings['styles'][] = array(
-			'css'                     => $theme_json->get_stylesheet( array( 'variables' ) ),
-			'__experimentalNoWrapper' => true,
-			'__unstableType'          => 'globalStyles',
+		$block_classes = array(
+			'css'            => 'styles',
+			'__unstableType' => 'theme',
```

Comment:
The plugin needs to be updated to account for the changes in `__unstableType` coming from core.

Reply:
https://github.com/WordPress/gutenberg/pull/36327